content/about/pricing.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Pricing
+weight: 4
+---
+
+Workers is now free to enable for all and are subject to some limits. 
+
+## Unlimited 
+To avoid the limits and enable KV, you can subscribe to the Workers Unlimited Plan for **$5 USD per month for a account**. Workers Unlimited plan is separate from any Cloudflare plan (Free, Professional, Business) you may have.
+
+If you are an Enterprise customer, reach out to your account team to confirm pricing details.
+
+## Limits 
+
+To protect our systems against abuse there are surge limits in place for domains without subscriptions. 
+
+Note only requests that hit a Worker script will be accounted for in the limits and in billing. Since Cloudflare Workers runs before the Cloudflare cache, the caching of a request still incurs costs. See definitions and behavior after a limit is hit for the free plan in the [limits article](/about/limits).
```

Comment:
Instead of "will be accounted for in the limits and in billing," consider "will count against your limits and your bill," which is more direct.

"the caching of a request still incurs costs" is unclear to me. Is there a way to be more explicit about what you mean?

I think you can remove "for the free plan" – that article also describes limits on the Unlimited plan, like CPU, memory usage, subrequests, etc.